### ISSUE TITLE
Add data tabs to GOlr pages

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/monarch-legacy#733]
Until we can implement logical or filters (see #723) we need to add data tabs generated from GOlr tables, similar to the production site.

**Comment by kshefchek** [comment]
So far I've added these to disease and phenotype pages.  Please note there are some bugs already documented in other issues.

http://beta.monarchinitiative.org/labs/widget-scratch/phenotype/HP:0002322
http://beta.monarchinitiative.org/labs/widget-scratch/disease/DOID:14330

**Comment by cmungall** [comment]
Nice! This is now ready to really allow us to explore data issues as well, as it's easy to navigate around.

I would say we should make these the main URLs. The legacy stuff could move to /legacy/ or similar

I'm sure you're aware of most of these and most are tracked elsewhere. I make no attempt to break these into display vs data
1. we still have to think of the best solution to override subject/object/subject-category column headings: new yaml vs override in code
2. page load time can still be a bit sluggish; need to investigate.
3. subject taxon: need this!! Chris is working on it...
4. the tab should show the number, and be greyed out if =0. Do this on client or server?
5. need a variant page
6. what's in the Gene tab on Phenotype page looks like diseases
7. no genotypes; e.g. http://beta.monarchinitiative.org/labs/widget-scratch/phenotype/MP:0013162
8. need to bring back Compare widget for any page that has a phenotype tab
9. Why is the self-hit in Compare not all filled in? [this is for me] http://beta.monarchinitiative.org/labs/widget-scratch/disease/OMIM:274600

**Comment by kltm** [comment]
For 1, I can almost guarantee the answer is the YAML for a host of reasons--remember that the description metadata in the files handles more than headers.

**Comment by kshefchek** [comment]
1. It could be sluggish due to calling fetchClassInfo and a Golr page on the server but this should only take 1-3 seconds.  I also noticed loading some of the js files can take some time.  I switched over to using the bbop.min.js file which helped.  Using the file caching may help, but this is disabled for beta.
2. Agreed, will make a new ticket
3. Will make a new ticket
4. This may be data related to https://github.com/monarch-initiative/dipper/issues/143
5. This could be a data issue or the filter configuration, will investigate.
6. So far I've just fixed for the disease page, do we need a compare tab on phenotype pages.  Working on fixing up gene and model pages.
7. Will investigate, perhaps we should break out into a ticket.
